kentikapi/internal/api_resources/alerting.go

```diff
@@ -0,0 +1,74 @@
+package api_resources
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_connection"
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_endpoints"
+	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_payloads"
+	"github.com/kentik/community_sdk_golang/kentikapi/models"
+)
+
+type AlertingAPI struct {
+	BaseAPI
+}
+
+// NewAlertingAPI is constructor
+func NewAlertingAPI(transport api_connection.Transport) *AlertingAPI {
+	return &AlertingAPI{
+		BaseAPI{Transport: transport},
+	}
+}
+
+func (a *AlertingAPI) CreateManualMitigation(ctx context.Context, mm models.ManualMitigation) error {
+	payload := api_payloads.ManualMitigationToPayload(mm)
+	var response api_payloads.CreateManualMitigationResponse
+
+	if err := a.PostAndValidate(ctx, api_endpoints.ManualMitigationPath, payload, &response); err != nil {
+		return err
+	}
+
+	if response.Response.Result != "OK" {
+		return errors.New("Creating Manual Mitigation Failed")
+	}
+
+	return nil
+}
+
+func (a *AlertingAPI) GetActiveAlerts(ctx context.Context, startTime time.Time, endTime time.Time, filterBy string, filterVal string, showMitigations bool,
```

Comment:
Please group alerts parameters under AlertsParameters data structure and pass it to GetActiveAlerts and GetAlertsHistory

Reply:
Done